tools/bin/go_core_tests

```diff
@@ -8,7 +8,7 @@ OUTPUT_FILE="./output.txt"
 echo "Failed tests and panics: ---------------------"
 echo ""
 GO_LDFLAGS=$(bash tools/bin/ldflags)
-go test -ldflags "$GO_LDFLAGS" -tags integration -p 3 -coverprofile=coverage.txt -covermode=atomic $1 | tee $OUTPUT_FILE
+go test -ldflags "$GO_LDFLAGS" -tags integration -p 3 -covermode=count -coverpkg=./... -coverprofile=coverage.txt  $1 | tee $OUTPUT_FILE
```

Comment:
Oh awesome, does `coverpkg` do exactly what we need?

Why did you change the mode from `atomic` to `count`? Is there a significant speedup?

Reply:
Thanks, I reverted to `atomic` as we have multi-threaded tests, and that one, though is slower, provides more accurate metrics.